less/nicer-btn-group.less

```diff
@@ -0,0 +1,23 @@
+.btn-group.id7-btn-group .btn-default.active {
```

Comment:
I was expecting these colours to come from the brand (and therefore default to something neutral before being overridden in branding.less). Either way, can we move the defaults into variables?